### PR TITLE
Add Windows support

### LINKED ISSUE #34: Windows support?
I am currently using the non-dev version on Windows, but I need PTX which seems to be included here. Is there some particular reason/complication for not supporting Windows right now?

### PR DESCRIPTION
**Pull request by nnkwame**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to 0 (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

Closes #34

This request adds windows installation support for the post-install script. The script extracts the contents of the cuda executable as described in the [installation guide](https://docs.nvidia.com/cuda/cuda-installation-guide-microsoft-windows/index.html#install-cuda-software).

### CODE REVIEW

**Comment by nnkwame** [comment]
@conda-forge-admin, please rerender

File: recipe/post-link.bat [hunk 1]
```diff
@@ -0,0 +1,2 @@
+python %PREFIX%\bin\cudatoolkit-dev-post-install.py
+if errorlevel 1 exit 1
```
Comment (xmnlab) [inline recipe/post-link.bat:2]: it is a good practice to have an empty line at the end of the file. In general, you can configure your IDE to do it for you automatically. but don't need to rebuild here to change that now.

**Review by xmnlab — APPROVED** [review]
LGTM! thanks for working on that!

**Comment by xmnlab** [comment]
@kidfisch do you want to add yourself as a maintainer here?

File: recipe/post-link.bat (hunk 1 above)
Comment (nnkwame) [reply recipe/post-link.bat:2]: Ok, thanks. Will incorporate this tip.

**Comment by nnkwame** [comment]
> @kidfisch do you want to add yourself as a maintainer here?

Yes, would be my pleasure to.

**Review by AgrawalAmey — APPROVED** [review]

**Comment by xmnlab** [comment]
thanks @kidfisch ! and feel free to open a new PR to add yourself as a maintainer! thanks!

**Pull request merged**